spec/lib/dfid/patch/specialist_publisher/themes_spec.rb

```diff
@@ -38,5 +38,14 @@
         'value' => 'Neglected%20Tropical%20Diseases'
       )
     end
+
+    it 'sorts themes alphabetically by label' do
+      patch.run
+
+      schema = JSON.parse(File.read(patch_location))
+      themes = schema['facets'].find { |facet| facet['key'] == 'theme' }
```

Comment:
These two lines are the same as the example above. I know how you feel about `let`, but moving them out will make both examples less noisy.